cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -321,13 +279,83 @@
                 border-bottom: 1px solid @gray-50;
             }
 
-            &-list:last-child .o-mega-menu_subcontent-item:last-child {
-                border-bottom: none;
-            }
+            &-link,
+            &-overview-link {
+                position: relative;
+                // Colors for :link, :visited, :hover, :focus, :active.
+                .u-link__colors( @pacific, @pacific, @pacific-60, @pacific, @dark-navy );
 
-            &-link {
                 padding-top: @grid_gutter-width / 2;
                 padding-bottom: @grid_gutter-width / 2;
+
+                &__has-children:after {
+                    .cf-icon();
+
+                    position: absolute;
+                    right: 0;
+                    top: 50%;
+                    transform: translateY(-50%);
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/7a83a6c0077c044326ca9a15cdffe249b7709a59